pom.xml

```diff
@@ -1,3 +1,189 @@
+					<value>true</value>
+				</property>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-gpg-plugin</artifactId>
+						<version>1.6</version>
+						<configuration>
+							<useAgent>true</useAgent>
+						</configuration>
+						<executions>
+							<execution>
+								<id>sign-artifacts</id>
+								<phase>verify</phase>
+								<goals>
+									<goal>sign</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+	</profiles>
+
+	<modules>
+		<module>languagetool-core</module>
+		<module>languagetool-language-modules/en</module>
+		<module>languagetool-language-modules/fa</module>
+		<module>languagetool-language-modules/fr</module>
+		<module>languagetool-language-modules/de</module>
+		<module>languagetool-language-modules/pl</module>
+		<module>languagetool-language-modules/ca</module>
+		<module>languagetool-language-modules/it</module>
+		<module>languagetool-language-modules/br</module>
+		<module>languagetool-language-modules/nl</module>
+		<module>languagetool-language-modules/pt</module>
+		<module>languagetool-language-modules/ru</module>
+		<module>languagetool-language-modules/ast</module>
+		<module>languagetool-language-modules/be</module>
+		<module>languagetool-language-modules/zh</module>
+		<module>languagetool-language-modules/da</module>
+		<module>languagetool-language-modules/eo</module>
+		<module>languagetool-language-modules/gl</module>
+		<module>languagetool-language-modules/el</module>
+		<module>languagetool-language-modules/is</module>
+		<module>languagetool-language-modules/ja</module>
+		<module>languagetool-language-modules/km</module>
+		<module>languagetool-language-modules/lt</module>
+		<module>languagetool-language-modules/ml</module>
+		<module>languagetool-language-modules/ro</module>
+		<module>languagetool-language-modules/sk</module>
+		<module>languagetool-language-modules/sl</module>
+		<module>languagetool-language-modules/es</module>
+		<module>languagetool-language-modules/sv</module>
+		<module>languagetool-language-modules/ta</module>
+		<module>languagetool-language-modules/tl</module>
+		<module>languagetool-language-modules/uk</module>
+		<module>languagetool-language-modules/de-DE-x-simple-language</module>
+		<module>languagetool-language-modules/all</module>
+		<module>languagetool-gui-commons</module>
+		<module>languagetool-commandline</module>
+		<module>languagetool-standalone</module>
+		<module>languagetool-office-extension</module>
+		<module>hunspell-native-libs</module>
+		<module>languagetool-wikipedia</module>
+		<module>languagetool-server</module>
+		<module>languagetool-http-client</module>
+		<module>languagetool-tools</module>
+		<module>languagetool-dev</module>
+		<!-- don't add languagetool-client-example here, it's built manually only -->
+		<module>languagetool-rpm-package</module>
+	</modules>
+=======
```

Comment:
This looks like a syntax error. The fact that there's so many changes shown here against pom.xml seems to indicate some issue with the way the PR was created, maybe?